### PR TITLE
File storage endpoint changes

### PR DESCRIPTION
**Pull request by roiaharon-pepperi**
added missing endpoint functionality

### CODE REVIEW

**Review by Chasky-H — CHANGES_REQUESTED** [review]

File: endpoints/fileStorage.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { IterableEndpoint } from '../endpoint';
+import { FileStorage, TempUrlResponse } from '../entities';
+import { PapiClient } from '../papi-client';
+
+export class FileStorageEndpoint extends IterableEndpoint<FileStorage> {
```
[the remark this answers is not in the document]
Comment (roiaharon-pepperi) [reply endpoints/fileStorage.ts:5]: why? the /file_storage endpoint support pagination, you'll need the iter function to go through all the files.

[the remark this answers is not in the document]
Comment (roiaharon-pepperi) [reply endpoints/fileStorage.ts:5]: you are right.. my mistake

File: endpoints/index.ts [hunk 2]
```diff
@@ -3,3 +3,4 @@ export * from './codeJobs';
 export * from './meta-data';
 export * from './maintenance';
 export * from './audit-logs';
+export * from './fileStorage';
```
[the remark this answers is not in the document]
Comment (roiaharon-pepperi) [reply endpoints/index.ts:6]: done

File: endpoints/fileStorage.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    constructor(service: PapiClient) {
+        super(service, '/code_jobs');
+    }
+
+    async name(fileName: string): Promise<FileStorage> {
```
[the remark this answers is not in the document]
Comment (roiaharon-pepperi) [reply endpoints/fileStorage.ts:10]: no.. according to api documentation, only /filestorage endpoint support POST

File: endpoints/fileStorage.ts [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import { IterableEndpoint } from '../endpoint';
+import { FileStorage, TempUrlResponse } from '../entities';
+import { PapiClient } from '../papi-client';
+
+export class FileStorageEndpoint extends IterableEndpoint<FileStorage> {
+    constructor(service: PapiClient) {
+        super(service, '/code_jobs');
```
[the remark this answers is not in the document]
Comment (roiaharon-pepperi) [reply endpoints/fileStorage.ts:7]: copy&paste problem.. 👎

**Review by Chasky-H — APPROVED** [review]

**Pull request merged**